client/src/components/layouts/certification.tsx

```diff
@@ -1,47 +1,48 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment, Component } from 'react';
 import { connect } from 'react-redux';
 
 import { fetchUser, isSignedInSelector, executeGA } from '../../redux';
 import { createSelector } from 'reselect';
 import Helmet from 'react-helmet';
 
+interface CertificationProps {
+  children?: React.ReactNode;
+  executeGA?: (args: { type: string; data: string }) => void;
+  fetchUser: () => void;
+  isSignedIn?: boolean;
+  pathname: string;
+}
+
 const mapStateToProps = createSelector(isSignedInSelector, isSignedIn => ({
   isSignedIn
 }));
 
 const mapDispatchToProps = { fetchUser, executeGA };
 
-class CertificationLayout extends Component {
+class CertificationLayout extends Component <CertificationProps> {
```

Comment:
```suggestion
class CertificationLayout extends Component<CertificationProps> {
```
It's nitpicky, but this is what the linter wants. :)